### ISSUE TITLE
Failed to fetch dynamically imported module

### ISSUE THREAD

**Issue by ZeRego** [lightdash/lightdash#8806]
### Description

**TLDR: frontend get out of date compared to deployed assets and server. This might cause a problem when requesting assets that no longer exist or connecting to the API. This is fixed if the user refresh the page** 

There is a problem with web workers in the FE when pivoting data.
When we make a new build/deployment the file path changes. Users who had a Lightdash page open before the deployment are still pointing to the old path which is not available anymore. 

<img width="697" alt="Screenshot 2024-01-31 at 17 37 08" src="https://github.com/lightdash/lightdash/assets/9117144/1e23467d-3603-4f51-b325-5ed43d3852d7">

```
// packages/frontend/src/hooks/tableVisualization/useTableConfig.ts
const createWorker = createWorkerFactory(
    () => import('../pivotTable/pivotQueryResults'),
);
```

### App Version

0.970.0

**Comment by ZeRego** [comment]
This error still appears if the user ignores the update notification and doesn't refresh the page.

We should capture this error and show an error message saying the user needs to refresh the page.
